libs/features/texts/src/acore-text/select-acore-string.component.html

```diff
@@ -0,0 +1,62 @@
+<keira-top-bar [selected]="handlerService.selected" [selectedName]="handlerService.selectedName" [isNew]="handlerService.isNew" />
+
+<div class="container-fluid">
+  <div class="content-block">
+    <keira-create
+      [entityTable]="entityTable"
+      [entityIdField]="entityIdField"
+      [customStartingId]="customStartingId"
+      [handlerService]="handlerService"
+      [queryService]="queryService"
+    />
+  </div>
+
+  <div class="content-block">
+    <p class="lead" [translate]="'SELECT_EXISTING'"></p>
+    <form [formGroup]="selectService.queryForm" class="mb-2">
+      <div class="row">
+        <ng-container [formGroup]="selectService.fields">
+          <div class="form-group col-sm-6 col-md-4 col-lg-3 col-xl-2">
+            <input [formControlName]="'entry'" type="number" class="form-control form-control-sm" id="search-id" placeholder="Entry" />
+          </div>
+          <div class="form-group col-sm-12 col-md-8 col-lg-6 col-xl-2">
+            <input
+              [formControlName]="'content_default'"
+              class="form-control form-control-sm"
+              id="search-default"
+              placeholder="Content Default"
+            />
+          </div>
+        </ng-container>
+        <div class="form-group col-sm-6 col-md-4 col-lg-3 col-xl-2">
+          <input [formControlName]="'limit'" class="form-control form-control-sm" id="limit" placeholder="Limit" />
+        </div>
+        <div class="mb-2 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+          <button id="search-btn" class="btn btn-primary btn-sm" [disabled]="selectService.queryForm.invalid" (click)="onSearch()">
+            <i class="fa fa-search"></i> {{ 'SEARCH' | translate }}
+          </button>
+        </div>
+      </div>
+      <keira-highlightjs-wrapper [code]="selectService.query" />
+    </form>
+
+    @if (selectService.rows) {
+      <div>
+        <ngx-datatable
+          class="bootstrap table table-striped text-center datatable-select"
+          [rows]="selectService.rows"
+          [headerHeight]="DTCFG.headerHeight"
+          [footerHeight]="DTCFG.footerHeight"
+          [columnMode]="DTCFG.columnMode"
+          [rowHeight]="DTCFG.rowHeight"
+          [limit]="DTCFG.limit"
+          [selectionType]="DTCFG.selectionType"
+          (select)="selectService.onSelect($event)"
+        >
+          <ngx-datatable-column name="entry" prop="entry" [minWidth]="80"></ngx-datatable-column>
+          <ngx-datatable-column name="content_default" prop="content_default" [minWidth]="400"></ngx-datatable-column>
```

Comment:
Since Angular 16, components without an inner content can have self-closing tags :)

```suggestion
          <ngx-datatable-column name="entry" prop="entry" [minWidth]="80" />
          <ngx-datatable-column name="content_default" prop="content_default" [minWidth]="400" />```